samples/Gonda/Gonda.Client/Gonda.Client.csproj

```diff
@@ -0,0 +1,22 @@
+<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <OutputType>Exe</OutputType>
+    <TargetFramework>net5.0</TargetFramework>
+  </PropertyGroup>
+  
+  <ItemGroup>
+    <Protobuf Include="..\Gonda.Server\Protos\greet.proto" GrpcServices="Client" />
+  </ItemGroup>
+
+  <ItemGroup>
+    <PackageReference Include="Google.Protobuf" />
+    <PackageReference Include="Grpc.Net.Client" />
+    <PackageReference Include="Grpc.Tools">
+      <PrivateAssets>all</PrivateAssets>
+      <IncludeAssets>runtime; build; native; contentfiles; analyzers; buildtransitive</IncludeAssets>
+    </PackageReference>
+    <PackageReference Include="Newtonsoft.Json" />
```

Comment:
Instead of using JSON.NET, you may want to use the IdentityModel client, which should simply things a bit (it's already used in another client sample so feel free to take a look)

Reply:
It doesn't seem like `IdentityModel.OidcClient` supports the Password flow... it's wanting me to add an AuthorizeUrl 🙃

Reply:
Woops, sorry, I should have been clearer: the method you're looking for is in the `IdentityModel` package, under the `TokenClient` class: https://github.com/IdentityModel/IdentityModel/blob/main/src/Client/TokenClient.cs#L110 😃

Reply:
aha! That one makes more sense... lol. ^_^